client/src/components/Applications/AppForm/AppForm.js

```diff
@@ -187,16 +187,12 @@ export const AppSchemaForm = ({ app }) => {
     initialValues.allocation = app.scheduler;
   }
 
-  // local state for alerts
-  const [visible, setVisible] = useState(true);
-  const onDismiss = () => setVisible(false);
-
   return (
     <div id="appForm-wrapper">
       {jobSubmission.response && (
         <div id="appForm-alerts">
           {jobSubmission.error ? (
-            <Alert color="warning" isOpen={visible} toggle={onDismiss}>
+            <Message type="warn" className="appDetail-error">
```

Comment:
@tacc-wbomar do we want these to be message type `error` and `success` respectively?

Reply:
Also @tacc-wbomar the new message bar is missing a dismiss or "x" button. Just wanted to check that this is also acceptable.

Reply:
The Message component does not have this feature. The message component styles are also not complete. I was unaware of this ticket, and not enough design work is complete for this ticket to begin. Please, see [FP-639](https://jira.tacc.utexas.edu/browse/FP-639). I will add, to FP-639, requirements to support the equivalent functionality of `isOpen` and `toggle`.

Reply:
And yes, they would be "error" and "success" respectively.

Reply:
I'm gonna try to perform FP-639, today. It's a lot of words, and I think I may be more prepared to make design decisions, or go back and forth with them on these styles. I may not start until late morning. If this delay is unacceptable, please proceed as best works for your team. Because as long as FP-639 is done before the next deploy, this work is fine.

Reply:
Okay. PR #260 provides everything this thread mentioned was needed.